src/App.tsx

```diff
@@ -1,16 +1,121 @@
-import React from 'react';
-import './App.css';
+import React, { FormEvent, useState } from 'react';
+import './App.scss';
 
-import users from './api/users';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { TodoList } from './components/TodoList/TodoList';
+
+const maxId = () => {
```

Comment:
```suggestion
const getMaxId = () => {
```

Reply:
You should get maxId from todos which place in state, and make it inside functional setState or pass todos[] in args.  And why you get new todo Id from `usersFromServer `